tests/js/common/shell/shell-keygen.js

```diff
@@ -273,7 +273,9 @@ function TraditionalSuite () {
         lastKey = key;
       }
 
-      assertEqual(lastKey, c.properties().keyOptions.lastValue);
+      if (!cluster && !cluster.isCluster && !cluster.isCluster()) {
```

Comment:
Should this be `||`s?

Reply:
The problem with making this an `||` is that the `if` condition will evaluate to true in the arangosh connected to a cluster. But in this case there is no `lastValue` attribute present.
So technically the `if` condition is correct, it's just not intuitive, because it is not checking if we are not running in cluster mode but if we are not running in cluster mode inside arangod.

Reply:
I mean, I guess I'm just confused about the logic of the statement altogether. If `!cluster` is true, then `cluster.isCluster` should be `undefined`, which automatically makes the rest of it true. So could you just simplify to `!cluster`?